dsn.go

```diff
@@ -81,7 +87,9 @@ type Config struct {
 	TokenAccessor    TokenAccessor // Optional token accessor to use
 	KeepSessionAlive bool          // Enables the session to persist even after the connection is closed
 
-	PrivateKey *rsa.PrivateKey // Private key used to sign JWT
+	PrivateKey       *rsa.PrivateKey // Private key used to sign JWT
```

Comment:
What do you mean specifically? Because in go driver there is a mix of both approaches - line indents are made of tabs, whereas field types in structs are aligned in spaces. 
If I decided myself, I would remove this spaces between field name and type :)

Reply:
There is no code change in this line. So could you please discard your changes? In some editor Tab is replaces by spaces. If you added spaces please remove it. So for readability of code it should be consistent with existing code.

Reply:
Yes, but take a broader look on this file :) There are multiple sections in the `Config` struct, like timeouts section, OCSP section, JWT section etc. And in each section type names are aligned to the longest variable name in this particular section. I introduced longer variable name in JWT section, so I needed to align remaining variables in this section. Please, take a look at the image attached.
![image](https://github.com/snowflakedb/gosnowflake/assets/132933642/a30f8d8a-837b-46b8-a7f6-991b0d0131c1)
Do you see my point?

Reply:
ok. sounds good.